configure

```diff
@@ -9511,6 +9511,51 @@ _ACEOF
 
 fi
 	# Dynamic linking for HP-UX
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for uuid_generate_time_safe in -luuid" >&5
+$as_echo_n "checking for uuid_generate_time_safe in -luuid... " >&6; }
+if ${ac_cv_lib_uuid_uuid_generate_time_safe+:} false; then :
+  $as_echo_n "(cached) " >&6
+else
+  ac_check_lib_save_LIBS=$LIBS
+LIBS="-luuid  $LIBS"
```

Comment:
`setup.py` has a bit of code to find `libuuid` in the proper place (`self.compiler.find_library_file(lib_dirs, 'uuid')`). It sounds like you should replicate that in the configure script, but it looks like you don't?